src/main/resources/CU-ApplicationResources.properties

```diff
@@ -584,6 +584,7 @@ error.paymentworks.new.vendor.custom.field.conversion.exception={0} generated th
 error.paymentworks.new.vendor.detail.not.returned.by.web.service=No detail data was obtained from PaymentWorks web service call for PaymentWorks Vendor ID {0}.
 error.paymentworks.duplicate.new.vendor.request=Duplicate PaymentWorks Vendor ID detected for New Vendor Request.
 error.paymentworks.intial.staging.table.save.failed=Initial attempt to save PaymentWorks New Vendor Request to staging table failed.
+error.paymentworks.paymentworksvendor.initial.save=Unable to save new Payment Works Vendor from Payment Works request.
```

Comment:
Is this the correct description to be using for the associated error? This seems to be occurring when the vendor could not be retrieved from PaymentWorks, so perhaps the message should be updated to reflect that.